fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -174,30 +174,33 @@ public static ColumnStatistic fromResultRow(ResultRow row) {
             String min = row.get(10);
             String max = row.get(11);
             if (min != null && !min.equalsIgnoreCase("NULL")) {
-                min = new String(Base64.getDecoder().decode(min),
-                            StandardCharsets.UTF_8);
-
-                try {
-                    columnStatisticBuilder.setMinValue(StatisticsUtil.convertToDouble(col.getType(), min));
-                    columnStatisticBuilder.setMinExpr(StatisticsUtil.readableValue(col.getType(), min));
-                } catch (AnalysisException e) {
-                    LOG.warn("Failed to deserialize column {} min value {}.", col, min, e);
+                min = new String(Base64.getDecoder().decode(min), StandardCharsets.UTF_8);
+                if (catalogId != 0 && min.equalsIgnoreCase("NULL")) {
```

Comment:
Why checking `catalogId != 0` here?
Do you mean to check if this is an internal catalog?
if yes, 2 questions:
1. Why need to check if this is an internal catalog?
2. Better use `catalogId != InternalCatalog.INTERNAL_CATALOG_ID`

Reply:
Yes, this to check is an internal catalog or not. Internal catalog get the min/max value using a separate SQL, and the value is already encoded by base64. Need to handle internal and external catalog separately.